### ISSUE TITLE
Custom Discord notification embed

### ISSUE THREAD

**Issue by ririko5834** [louislam/uptime-kuma#1388]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

New Notification

### 🔖 Feature description

Add way to setup custom Discord notification body with placeholders

### ✔️ Solution

Add some textfield where you can edit JSON body that will be posted to webhook for providers such as Discord, Slack, and others.
There should be placeholders like `{{monitor.name}}` that will be replaced with monitor name, etc.

**Comment by Serpensin** [comment]
Yeah. I need the ability to remove the 'Service URL' for specific monitors because they contain sensible data.

**Comment by xeruf** [comment]
See https://github.com/louislam/uptime-kuma/issues/907

**Comment by MWGMorningwood** [comment]
> Yeah. I need the ability to remove the 'Service URL' for specific monitors because they contain sensible data.

^this- a lot of the queries i'm using now are API based and include the key

**Comment by ToxykAuBleu** [comment]
Hello 👋 any news ?
Currently, full customization of message format is awesome, _**but**_, it lacks embeds support (like the normal one) 😕
